extensions/wikia/Email/i18n/TwitchDisconnect.i18n.php

```diff
@@ -0,0 +1,94 @@
+	'emailext-twitchdisconnect-instructions' => 'Si previamente estableciste una contraseña en FANDOM, ninguna acción es necesaria. Puedes ignorar este mensaje y continuar ingresando FANDOM con tu antigua contraseña. De lo contrario, haz clic en el enlace de arriba para establecer su nueva contraseña en FANDOM.',
+);
+
+$messages['fr'] = array(
+	'emailext-twitchdisconnect-subject' => 'Compte FANDOM déconnecté de Twitch !',
+	'emailext-twitchdisconnect-summary' => "Il semble que vous ayez déconnecté votre compte FANDOM de Twitch. Nous valorisons votre place au sein de la communauté FANDOM, c'est pourquoi nous avons conservé votre historique, vos messages et les détails de votre profil.",
+	'emailext-twitchdisconnect-intro' => "Vous pouvez utiliser le même nom d'utilisateur, mais vous devrez peut-être cliquer sur le lien ci-dessous pour définir votre nouveau mot de passe sur FANDOM :",
+	'emailext-twitchdisconnect-instructions' => "Si vous avez déjà défini un mot de passe sur FANDOM, aucune action n'est requise de votre part. Vous pouvez ignorer cet e-mail et continuer à vous connecter à FANDOM avec votre ancien mot de passe. Dans le cas contraire, veuillez cliquer sur le lien ci-dessus pour définir un nouveau mot de passe sur FANDOM.",
+);
+
+$messages['it'] = array(
+	'emailext-twitchdisconnect-subject' => 'Il tuo account FANDOM adesso non è più collegato a Twitch!',
+	'emailext-twitchdisconnect-summary' => 'Sembra che tu abbia appena scollegato il tuo account FANDOM da Twitch. Abbiamo mantenuto tutti i tuoi contributi, messaggi e il tuo profilo dal momento che sei una parte importante della comunità di FANDOM.',
+	'emailext-twitchdisconnect-intro' => 'È possibile usare lo stesso nome utente, ma potrebbe essere necessario fare clic sul link qui di seguito per impostare la tua nuova password su FANDOM:',
+	'emailext-twitchdisconnect-instructions' => "Se hai già impostato una password su FANDOM, non occorre fare altro. Puoi ignorare questo messaggio e continuare l'accesso a FANDOM con la vecchia password. In caso contrario, fai clic sul link qui sopra per impostare la tua nuova password su FANDOM.",
+);
+
+$messages['ja'] = array(
+	'emailext-twitchdisconnect-subject' => 'FANDOMアカウントとTwitchとの連携が解除されました',
+	'emailext-twitchdisconnect-summary' => 'FANDOMアカウントとTwitchアカウントの連携が解除されました。FANDOMコミュニティ上での履歴・メッセージ・プロフィールなど、大切な情報はすべて保持されていますので、ご安心ください。',
+	'emailext-twitchdisconnect-intro' => '同じユーザー名を継続して使用する場合、以下のリンクから新しいパスワードを設定する必要があります。',
+	'emailext-twitchdisconnect-instructions' => '該当のアカウントでパスワードを設定している場合、再設定の必要はありません。引き続き設定済みのパスワードでFANDOMをご利用いただけます。パスワードの設定を行っていない場合は、お手数ですが上記のリンクからパスワードを設定してください。',
+);
+
+$messages['ko'] = array(
+	'emailext-twitchdisconnect-instructions' => '이미 위키아 비밀번호를 설정하셨다면 걱정하지 마세요! 귀하의 계정은 안전합니다. 이 이메일은 무시하시고 기존의 비밀번호로 로그인하시면 됩니다. 그 외의 경우에는 임시 비밀번호로 로그인한 뒤 환경 설정에서 원하는 비밀번호로 설정을 변경하시면 됩니다.',
+	'emailext-twitchdisconnect-subject' => '위키아 계정의 페이스북 연동이 해제되었습니다',
```

Comment:
Google Translate claims this means `The Facebook link of the Wikia account has been canceled`. Shouldn't we just wait for translations from crowdin?

Reply:
Sorry my Korean is not too good... ;P